### ISSUE TITLE
CI failing: HTTP failure with marker link check

### ISSUE THREAD

**Issue by hasbro17** [operator-framework/operator-sdk#2741]
The markdown tests job is currently failing for all(most?) PRs that don't seem to have anything to do with the failure e.g #2511 #2740 
https://travis-ci.org/github/operator-framework/operator-sdk/jobs/667925856#L1052
```console
./hack/ci/marker -e website
HTTP failure           (Operators -> https://coreos.com/operators/) at ./README.md:11 : received fatal alert: ProtocolVersion
HTTP failure           (introduction blog post -> https://coreos.com/blog/introducing-operator-framework) at ./README.md:9 : received fatal alert: ProtocolVersion
Makefile:203: recipe for target 'test-markdown' failed
make: *** [test-markdown] Error 1
The command "make test-sanity test-unit test-markdown" exited with 2.
```

/cc @joelanford @estroz @camilamacedo86

**Comment by hasbro17** [comment]
Also @asmacdo if you've ever seen this failure for the website docs before.

**Comment by camilamacedo86** [comment]
Hi @hasbro17 shows be at ./README.md:11 so I do not think that has relation with the website dir at all.

**Comment by estroz** [comment]
Meta: need to investigate running marker only on docs that have been changed in a PR to avoid rate limiting (429 error), and a nightly/"merge with master" job to check all links to catch errors like this. Also need to report health of master when "merge with master" tests fail with a notification.

/priority important-longterm

**Comment by hasbro17** [comment]
Now hitting:
```
Running markdown link checker against changed markdown files
Found broken url       (Upgrade  -> https://github.com/kubernetes-sigs/controller-tools) at ./CHANGELOG.md:165 : 429 Too Many Requests
Found broken url       (Upgrade  -> https://github.com/kubernetes-sigs/controller-tools) at ./CHANGELOG.md:229 : 429 Too Many Requests
Makefile:203: recipe for target 'test-markdown' failed
make: *** [test-markdown] Error 1
```

 Hopefully that's just temporary but adding this here for future reference.

**Comment by estroz** [comment]
Closed by #3048
